docs/topics_5/proc_web-troubleshooting-common-issues.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
+//
+// * docs/web-console-guide_5/master.adoc
+
+[id="proc_web-troubleshooting-common-issues_{context}"]
+= Common issues when troubleshooting a {WebName} installation on OpenShift Container Platform
+
+.`NoRouteToHostException` error
+
+The following error might appear in the `mta-web-console-executor` pod log:
+
+----
+13:44:03,501 SEVERE [org.jboss.windup.web.messaging.executor.ExecutorBootstrap] (main) Could not start messaging listener due to: Failed to connect to any server. Servers tried: [http-remoting://192.0.2.4:8080 (java.net.NoRouteToHostException: No route to host)]: javax.naming.CommunicationException: Failed to connect to any server. Servers tried: [http-remoting://192.0.2.4:8080 (java.net.NoRouteToHostException: No route to host)]
+----
+
+This error message is displayed in a new deployment because the `mta-web-console-executor` pod starts before the `mta-web-console` pod. Wait for the `mta-web-console` pod to start.
+
+.A pod reports insufficient resources
+
+After attempting to deploy {WebNameTitle} on OpenShift, a pod is unable to start, and the following error is seen in the `Events` tab:
```

Comment:
Second comma is unnecessary. Remove it.
"is seen" > "is displayed"
"After attempting to ..." . Fluff. The whole section is about deploying {WebName} (**not** {WebNameTitle}, which is for the title of the doc).
It is not "a pod". It is the web console pod. So you can change it to this: `The `mta-web-console` pod is not running and the following error is displayed in the *Events* tab.`